packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -642,11 +502,94 @@ export class Relayer {
         } else throw receipt;
       } else throw receipt;
     } catch (error) {
-      this.logger.error({ at: "InsuredBridgeRelayer#Disputer", message: "Something errored disputing relay!", error });
+      this.logger.error({ at: "AcrossRelayer#Disputer", message: "Something errored disputing relay!", error });
     }
   }
 
-  private generateSlowRelayTx(deposit: Deposit, realizedLpFeePct: BN): TransactionType {
+  private async _processTransactionBatch(
+    transactions: { transaction: TransactionType | any; message: string; mrkdwn: string }[]
```

Comment:
nit: Why did we need to add `| any` (just curious -- I know this is sometimes necessary to get things to compile).

Reply:
reason is two fold: when we dont have transactions to append we use `null` and secondly the `._parent.*` props, used later on, are not included in the `TransactionType`.

Reply:
Can you extend the `TransactionType` to include those other props to avoid using `any`?

Reply:
yup. Let me get that right.